core/src/main/scala/kafka/tools/PerfConfig.scala

```diff
@@ -40,20 +40,5 @@ class PerfConfig(args: Array[String]) {
   val showDetailedStatsOpt = parser.accepts("show-detailed-stats", "If set, stats are reported for each reporting " +
```

Comment:
@ijuma Did you mean we need to merge two ConsumerPerfConfig and use only one of them?